src/main/kotlin/uk/gov/justice/digital/hmpps/incentivesapi/service/NextReviewDateService.kt

```diff
@@ -49,8 +60,12 @@ class NextReviewDateService(private val input: NextReviewDateInput) {
   }
 
   private fun wasConfirmedBasic(): Boolean {
-    val (iepDetails) = input
+    val iepDetails = input.iepDetails
 
     return isOnBasic() && iepDetails.size >= 2 && iepDetails[1].iepLevel == BASIC
   }
+
+  private fun isNewPrisoner(): Boolean {
+    return input.iepDetails.isEmpty()
```

Comment:
i have a sneaking suspicion that we might want to instead check that `iepDetails` only has not-real-review-type rows cos there will be an automatic one created at induction

Reply:
Yeah, I think this is simple and nice at the moment but I saw another ticket about only considering "real reviews".

Ultimately, we're iterating this to get to a more accurate next review date.

Reply:
hmm can't see how `iepDetails` is could ever be empty. given that this is calculating the next review date to put into a response that returns someone's *current incentive level*, the code path can't reach here without having found _some_ records from the past.